src/System.Private.CoreLib/shared/System/Number.BigInteger.cs

```diff
@@ -27,14 +45,16 @@ internal static partial class Number
                 10000000,   // 10^7
             };
 
-            private static readonly int[] s_s_Pow10BigNumTableIndices = new int[]
+            private static readonly int[] s_Pow10BigNumTableIndices = new int[]
             {
                 0,          // 10^8
                 2,          // 10^16
                 5,          // 10^32
                 10,         // 10^64
                 18,         // 10^128
                 33,         // 10^256
+                61,         // 10^512
+                116,        // 10^1024
             };
 
             private static readonly uint[] s_Pow10BigNumTable = new uint[]
```

Comment:
Is it feasible/worthwhile to have debug-only code that validates the entries in this table don't contain a mistake? I didn't see any

Reply:
Yes, we have that elsewhere.